book/templates/chapter-template/README.md

```diff
@@ -0,0 +1,138 @@
+# Welcome
+
+Well done on taking the first steps towards writing a chapter for _The Turing Way_  book!
+
+In order to standardise the book's content and maintain consistency throughout, we have created this template to guide you as you write the content for your chapter.
+
+## Before You Begin
+
+_The Turing Way_ book is made up of Guides which span several chapters.
+This means that when your chapter is added to the book, it will belong to one of its Guides.
+
+Structurally, a Guide in _The Turing Way_ Github repository is a folder and your chapter is a sub-folder within it.
+All Guides in _The Turing Way_ live in the `book/website` directory.
+
+If your chapter belonged to Guide 1, for example, then the file structure could resemble the following:
+
+```
+book\website
+│
+└───Guide 1
+│   │   Guide 1's Landing Page.md
+│   │   Your Chapter's Landing Page.md
+│   │
+│   └───Your Chapter
+│       │   Your Chapter's Topic (1).md
+|       |   Your Chapter's Topic (2).md
+|       |   Your Chapter's Topic (3).md
+|       |   ...
+│       │   Your Chapter Checklist.md
+│       │   Your Chapter Resources.md
+│   
+└───Guide 2
+    │   Guide 2's Landing page.md
+    │   ...
+```
```

Comment:
Provide a real example here (from Reproducibility and/or community handbook) so that the file naming is also apparent.